sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,72 @@
+WITH flows AS (
+  -- last fxa_uid for each flow_id
+  SELECT
+    flow_id,
+    MIN(flow_started) AS flow_started,
+    ARRAY_AGG(fxa_uid IGNORE NULLS ORDER BY fxa_uid_timestamp DESC LIMIT 1)[
+      SAFE_OFFSET(0)
+    ] AS fxa_uid,
+  FROM
+    `moz-fx-cjms-nonprod-9a36`.cjms_bigquery.flows_v1
+  WHERE
+    -- only use the last 10 days in stage
+    submission_date >= CURRENT_DATE - 10
+  GROUP BY
+    flow_id
+),
+attributed_flows AS (
+  -- last flow_id in aic for each fxa_uid
+  SELECT
+    fxa_uid,
+    ARRAY_AGG(STRUCT(flow_id, flow_started) ORDER BY flow_started DESC LIMIT 1)[SAFE_OFFSET(0)].*,
+  FROM
+    flows
+  JOIN
+    EXTERNAL_QUERY("moz-fx-cjms-nonprod-9a36.us.cjms-sql", "SELECT flow_id FROM aic")
+  USING
+    (flow_id)
+  GROUP BY
+    fxa_uid
+),
```

Comment:
I think taking the last flow per FxA before knowing the subscription created cutoff might miss some edge cases.

For example, if someone subscribes to VPN and then later clicks a VPN affiliate link (or has a page with affiliate URL params reloaded/navigated in history), this would only include the later flow and the subscription won't be counted as attributed in the `attributed_subs` CTE below.

Reply:
I will pass that on, but this is the compromise I worked out with subplat eng

Reply:
good point, i'll discuss this with sarah bird and file a follow up if necessary

Reply:
will rework this to take the last flow after joining subscription